bitbots_body_behavior/src/bitbots_body_behavior/decisions/closest_to_ball.py

```diff
@@ -7,10 +7,12 @@
 class ClosestToBallNoGoalie(AbstractDecisionElement):
     def __init__(self, blackboard, dsd, parameters=None):
         super(ClosestToBallNoGoalie, self).__init__(blackboard, dsd, parameters)
+        self.use_time_to_ball = parameters.get("use_time", False)
```

Comment:
I don't know if we want to do this as a param. If this pr works well it should substitute the old approach completely.

Reply:
This adds code and complexity to the dsd and python files

Reply:
I think we can change it in https://github.com/bit-bots/bitbots_behavior/pull/209 once it it tested in game